exercises/flow-control/src/test/java/com/cbfacademy/FlowControlExercisesTest.java

```diff
@@ -63,7 +63,7 @@ public void oddAndEvenNumbers() {
         assertThat(result.size(), is(2));
         assertThat(keys.contains("SumOfEvens"), is(true));
         assertThat(keys.contains("SumOfOdds"), is(true));
-        assertThat(result.get("SumOfEvens"), is(2450));
+        assertThat(result.get("SumOfEvens"), is(2550));
```

Comment:
Will this be moved back to the main codebase?

Reply:
Yep, that's why it got its own commit 👍🏾